defaults/defaults.go

```diff
@@ -101,3 +101,20 @@ var (
 	// CiliumPodSelector is the pod selector to be used for the Cilium agents.
 	CiliumPodSelector = "k8s-app=cilium"
 )
+
+// all hubble values from `cilium-config“ configmap
+// https://github.com/cilium/cilium/blob/d9a04be9d714e5f5544cbca7ef8db7a151bfce96/install/kubernetes/cilium/templates/cilium-configmap.yaml#L709-L750
+var HubbleKeys = []string{
```

Comment:
Can you add a quick comment as to what this array is for? And, perhaps, a follow-up PR in the helm chart that points users to this code if they add more hubble config keys?

Reply:
Done. On second thought - a key is a hubble related key if and only if it has `hubble` substring in its name, at least up until now this was always true. Would it be less hacky if we just check if the string contains `hubble`?

Reply:
Bah, I'm not sure. @chancez?

Reply:
Yeah, I like that proposal. Feel free to implement. But the manually maintained list also is fine, I don't expect too much churn.

Reply:
I'm not sure to be honest. I'd have to review all the flags in more detail. I think if your checking only for `hubble` then you will also catch the `hubble-relay` options, so that might be an issue?

Reply:
Hubble Relay options are stored in a different configmap. This map here is only read by the Cilium Agent and the Operator (the latter does not parse any Hubble values).

Reply:
Ah I didn't read the whole context, that makes sense.